### PR TITLE
Update feature list for SIG/Network

### PR DESCRIPTION
**Pull request by lmbr-pip**
Provide an updated list.

### CODE REVIEW

File: releases/23.05.0/23050 feature list.md [hunk 1]
```diff
@@ -112,9 +112,10 @@ Metrics files should ideally be output to an active projects `<project-root>/use
 
 ## sig-network
 
-* Adds new warnings when client and server have differences in networked code and properties to aid debugging of these issues.
+* Provide a major update to the Multiplayer sample game: https://github.com/o3de/o3de-multiplayersample to provide a feature rich game experience with UX, audio, visual FX, player vs player and player vs environment elements.
+* Provide a new simple player spawner component.
```
Comment (lmbr-pip) [inline releases/23.05.0/23050 feature list.md:116]:
```suggestion
* Provide a new simple player spawner component that avoids the needs to write custom spawn logic, https://github.com/o3de/o3de/pull/13871
```

**Review by chanmosq — APPROVED** [review]
Thanks!

**Review by vincent6767 — APPROVED** [review]

**Review by AMZN-Olex — APPROVED** [review]

File: releases/23.05.0/23050 feature list.md [hunk 2]
```diff
@@ -112,9 +112,10 @@ Metrics files should ideally be output to an active projects `<project-root>/use
 
 ## sig-network
 
-* Adds new warnings when client and server have differences in networked code and properties to aid debugging of these issues.
+* Provide a major update to the Multiplayer sample game: https://github.com/o3de/o3de-multiplayersample to provide a feature rich game experience with UX, audio, visual FX, player vs player and player vs environment elements.
```
Comment (allisaurus) [inline releases/23.05.0/23050 feature list.md:115]:
```suggestion
* Provides a major update to the Multiplayer sample game: https://github.com/o3de/o3de-multiplayersample to provide a feature rich game experience with UX, audio, visual FX, player vs player and player vs environment elements.
```

File: releases/23.05.0/23050 feature list.md [hunk 3]
```diff
@@ -112,9 +112,10 @@ Metrics files should ideally be output to an active projects `<project-root>/use
...
+* Provide a new simple player spawner component that avoids the needs to write custom spawn logic, [#13871](https://github.com/o3de/o3de/pull/13871)
```
Comment (allisaurus) [inline releases/23.05.0/23050 feature list.md:116]:
```suggestion
* Provides a new simple player spawner component that avoids the needs to write custom spawn logic, [#13871](https://github.com/o3de/o3de/pull/13871)
```

File: releases/23.05.0/23050 feature list.md [hunk 4]
```diff
@@ -112,9 +112,10 @@ Metrics files should ideally be output to an active projects `<project-root>/use
...
+* Add a validation mechanism to identify when client and server have differences in networked code and properties to aid debugging of these issues.
```
Comment (allisaurus) [inline releases/23.05.0/23050 feature list.md:117]:
```suggestion
* Adds a validation mechanism to identify when client and server have differences in networked code and properties to aid debugging of these issues.
```

**Review by allisaurus — COMMENTED** [review]
Left a few suggested changes for consistent tense across bullets

**Review by kberg-amzn — APPROVED** [review]
Looks good! Thanks Pip! 👍

**Review by tonybalandiuk — APPROVED** [review]
looks good, thanks

**Pull request merged**